apps/web/components/AppListCard.tsx

```diff
@@ -2,7 +2,7 @@ import { useRouter } from "next/router";
 import type { ReactNode } from "react";
 import { useEffect, useRef, useState } from "react";
 import { z } from "zod";
-
+import classNames from "@calcom/lib/classNames";
```

Comment:
Prettier error in the file. Eslint should have fixed it automatically. May be your commit hooks and IDE isn't configured correctly.

Reply:
Yeah. It's correct now.